src/main/java/com/softech/coursesvc/controller/CourseController.java

```diff
@@ -1,17 +1,40 @@
 package com.softech.coursesvc.controller;
 
+import com.softech.coursesvc.CourseRepository;
+import com.softech.coursesvc.model.Course;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.RestController;
 
+import java.util.List;
+
 @RestController
 public class CourseController {
 
+    @Autowired
+    private CourseRepository courseRepository;
+
 
     //dummy endpoint
     @GetMapping("/hello")
     public String hello() {
         return "Hello!!!";
     }
 
+    @GetMapping("/v1/course/")
+       public List<Course> getAllCourses()
```

Comment:
I thi k heena was working on this GetAll endpoint.. you can remove this and see how she does it... we dont want to return List<Course> as i explianed in recording.. lisyen again for more details or see her Pr and discuss with her why we should not do this..